amundsen_application/static/js/components/common/SearchBar/InlineSearchResults/styles.scss

```diff
@@ -25,17 +25,19 @@
     }
   }
 
-  .inline-results-section {
-    padding-top: 8px;
-    padding-bottom: 8px;
+  .search-item-section {
+    padding-top: $spacer-1;
+    padding-bottom: $spacer-1;
+  }
 
+  .inline-results-section {
     &:not(:last-of-type) {
       border-bottom: 1px solid $stroke;
     }
 
     .section-title,
     .section-footer {
-      margin-left: 24px;
+      margin: $spacer-1 0 $spacer-1 $spacer-3;
```

Comment:
1. Visually the right margin won't show a difference because there is only empty space towards the right, but I will add it in for balance.

2. Will do